integration/util_test.go

```diff
@@ -664,8 +664,12 @@ func rebootContainerd(t *testing.T, sh *shell.Shell, customContainerdConfig, cus
 		t.Fatalf("failed to create pipe: %v", err)
 	}
 	reporter := testutil.NewTestingReporter(t)
-	testutil.FatalMonitor(reporter, outR, errR)
-	var m *testutil.RemoteSnapshotMonitor = testutil.NewRemoteSnapshotMonitor(reporter, outR, errR)
+	var m *testutil.LogMonitor = testutil.NewLogMonitor(reporter, outR, errR)
+
+	err = testutil.LogConfirmStartup(m)
+	if err != nil {
+		t.Fatalf("snapshotter startup failed: %v", err)
+	}
```

Comment:
`if err := testutil.LogConfirmStartup(m); err != nil { ...`

Reply:
Good catch. I will note this for a fix in a followup PR.